.github/workflows/sbom.yml

```diff
@@ -1,7 +1,7 @@
 name: SBOM
 on:
   push:
-    branches: ["master", "24.0", "23.3"]
+    branches: ["main", "24.0", "23.3"]
```

Comment:
There are more `master` references in cherryPick which is used in other projects, at some point it could fail, can we figure out somehow the name of the default branch?

Reply:
i will do that in another PR.